### ISSUE TITLE
Wallet fingerprint not added to metric

### ISSUE THREAD

**Issue by lyncos** [retzkek/chia_exporter#12]
When I query the /metrics endpoint I see all my wallet and balances, but the fingerprint values are missing. I had the issue with chia 1.2.3 and 1.2.5.

It also seems to miss all other wallets (only finding wallet_id=1 and I have many wallets)
```

chia_wallet_farmed_amount{wallet_fingerprint="",wallet_id="1"} 6.000006000021e+12
# HELP chia_wallet_fee_amount Fee amount amount
# TYPE chia_wallet_fee_amount gauge
chia_wallet_fee_amount{wallet_fingerprint="",wallet_id="1"} 6.000021e+06
# HELP chia_wallet_height Wallet synced height.
# TYPE chia_wallet_height gauge
chia_wallet_height{wallet_fingerprint="",wallet_id="1"} 825447
# HELP chia_wallet_last_height_farmed Last height farmed
# TYPE chia_wallet_last_height_farmed gauge
chia_wallet_last_height_farmed{wallet_fingerprint="",wallet_id="1"} 816184
# HELP chia_wallet_pool_reward_amount Pool Reward amount
# TYPE chia_wallet_pool_reward_amount gauge
chia_wallet_pool_reward_amount{wallet_fingerprint="",wallet_id="1"} 5.25e+12
# HELP chia_wallet_reward_amount Reward amount
# TYPE chia_wallet_reward_amount gauge
chia_wallet_reward_amount{wallet_fingerprint="",wallet_id="1"} 7.5e+11
# HELP chia_wallet_sync_status Sync status, 0=not synced, 1=syncing, 2=synced
# TYPE chia_wallet_sync_status gauge
chia_wallet_sync_status{wallet_fingerprint="",wallet_id="1"} 2
# HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.

```

Also it is timing out on:

2021/09/07 12:09:42 error calling get_wallet_balance: Post "https://localhost:9256/get_wallet_balance": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

In chia my wallets take some time to answer, I think that timeout is causing me problems. Can I increase it somewhere?

**Comment by retzkek** [comment]
Do you also see a timeout error for `get_public_keys`, that's the API endpoint that is used for wallet fingerprints? Timeouts are currently hardcoded @  https://github.com/retzkek/chia_exporter/blob/master/main.go#L91 but could be increased and/or made configuration options, at least the Client.Timeout, which is what you're hitting. 

It's not too hard to modify the hard-coded value and [build it yourself](https://github.com/retzkek/chia_exporter#building-and-running) in the meantime.

**Comment by retzkek** [comment]
I went ahead and added a `-timeout` option and released v0.5.2, give it a try, let me know if it doesn't solve your problem.
